executor/common_linux.h

```diff
@@ -469,7 +470,7 @@ static void netlink_add_neigh(struct nlmsg* nlmsg, int sock, const char* name,
 #endif
 #endif
 
-#if SYZ_EXECUTOR || SYZ_NET_DEVICES || SYZ_NET_INJECTION
+#if SYZ_EXECUTOR || SYZ_NET_DEVICES || SYZ_NET_INJECTION || SYZ_DEVLINK_PCI
```

Comment:
Can you fold the changes from "common/common_linux.h: addressed CI errors" into the relevant commits (i.e. amend with rebase -i etc.)?

Reply:
Done